docs/tf_model_binary_input.md

```diff
@@ -4,7 +4,7 @@ This guide shows how to convert TensorFlow models and deploy them with the OpenV
 
 - In this example TensorFlow model [ResNet](https://github.com/tensorflow/models/tree/v2.2.0/official/r1/resnet) will be used.
 
-- TensorFlow model can be converted into Intermediate Representation format using model_optimizer tool. There are several formats for storing TensorFlow model. In this guide, we present conversion from SavedModel format. More information about conversion process can be found in the [model optimizer guide](https://docs.openvino.ai/2023.2/openvino_docs_MO_DG_prepare_model_convert_model_tutorials.html).
+- TensorFlow model can be converted into Intermediate Representation format using model_optimizer tool. There are several formats for storing TensorFlow model. In this guide, we present conversion from SavedModel format. More information about conversion process can be found in the [model optimizer guide](https://docs.openvino.ai/2023.3/openvino_docs_MO_DG_prepare_model_convert_model_tutorials.html).
```

Comment:
This is deprecated in OV.  At some point in time we should redirect to:
https://docs.openvino.ai/2023.3/tutorials.html